### PR TITLE
ci: update pullapprove.yml to cover all files in the repo

### PR DESCRIPTION
**Pull request by josephperrott**
This change should be the final change needed before the verify script created in #35060 can pass.

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -534,6 +534,41 @@ groups:
...
+          'modules/benchmarks/**'
+          ])
+    reviewers:
+      users:
+        - IgorMinar
+        - kara
+        - pkozlowski-opensource
+      teams:
+        - ~framework-global-approvers
+        - ~framework-global-approvers-for-docs-only-changes
```
Comment (IgorMinar) [inline .pullapprove.yml]:
```suggestion
```

File: .pullapprove.yml [hunk 2]
```diff
@@ -534,6 +534,41 @@ groups:
...
+        contains_any_globs(files, [
+          'modules/playground/**',
+          ])
+    reviewers:
+      users:
+        - IgorMinar
+        - kara
+      teams:
+        - ~framework-global-approvers
+        - ~framework-global-approvers-for-docs-only-changes
```
Comment (IgorMinar) [inline .pullapprove.yml]:
```suggestion
```

File: .pullapprove.yml [hunk 3]
```diff
@@ -859,7 +896,10 @@ groups:
           'aio/content/examples/docs-style-guide/**',
           'aio/content/images/guide/docs-style-guide/**',
           'aio/content/guide/visual-studio-2015.md',
-          'aio/content/examples/visual-studio-2015/**'
+          'aio/content/examples/visual-studio-2015/**',
+          'packages/examples/README.md',
+          'packages/examples/index.html',
+          'packages/examples/tsconfig-e2e.json'
```
Comment (IgorMinar) [inline .pullapprove.yml]: these should have been dev-infra, no?

**Review by IgorMinar — APPROVED** [review]
lgtm after cleanup

File: .pullapprove.yml (hunk 3 above)
Comment (josephperrott) [reply .pullapprove.yml]: That makes more sense.

File: .pullapprove.yml [hunk 4]
```diff
@@ -534,8 +534,40 @@ groups:
         - ~framework-global-approvers
         - ~framework-global-approvers-for-docs-only-changes
 
+  # =========================================================
```
Comment (gkalpak) [inline .pullapprove.yml]:
```suggestion

  # =========================================================
```

:pray:

File: .pullapprove.yml [hunk 5]
```diff
@@ -534,8 +534,40 @@ groups:
...
+        - ~framework-global-approvers
 
   # =========================================================
+  #  Framework: Playground
+  # =========================================================
+  fw-playground:
+    conditions:
+      - >
+        contains_any_globs(files, [
+          'modules/playground/**',
```
Comment (gkalpak) [inline .pullapprove.yml]:
```suggestion
          'modules/playground/**'
```

(fonr consistency :grin:)

File: .pullapprove.yml [hunk 6]
```diff
@@ -534,8 +534,40 @@ groups:
...
+          ])
+    reviewers:
+      users:
+        - IgorMinar
+        - kara
+      teams:
+        - ~framework-global-approvers
+  # =========================================================
```
Comment (gkalpak) [inline .pullapprove.yml]:
```suggestion


  # =========================================================
```

File: .pullapprove.yml [hunk 7]
```diff
@@ -558,7 +590,6 @@ groups:
         - ~framework-global-approvers
         - ~framework-global-approvers-for-docs-only-changes
 
-
```
Comment (gkalpak) [inline .pullapprove.yml]: 😱 😢

File: .pullapprove.yml [hunk 8]
```diff
@@ -898,6 +946,7 @@ groups:
           'tools/ng_rollup_bundle/**',
           'tools/ngcontainer/**',
           'tools/npm/**',
+          'tools/pullapprove/**',
```
Comment (gkalpak) [inline .pullapprove.yml]: Move below `tools/public_api_guard/...` for alphabetic order 🙏

**Review by gkalpak — APPROVED** [review]
Nits ftw 😇

**Comment by josephperrott** [comment]
Merge Assistance: test_docs_examples flake is unrelated to this pullapprove config change.

**Pull request closed without merging**